src/Raven.Client/Documents/BulkInsert/BulkInsertOperation.cs

```diff
@@ -683,7 +683,7 @@ public class TimeSeriesBulkInsert : IDisposable
             private readonly string _id;
             private readonly string _name;
             private bool _first = true;
-            private const int _maxTimeSeriesInBatch = 128;
+            private const int _maxTimeSeriesInBatch = 1024;
```

Comment:
I still think we can do better by letting the server decide on the batch size (and not the client like here).
So, should we keep it as a const so we can easily remove it if needed, or pass it as a parameter?

Reply:
Let's do that as a parameter, yes.

Reply:
done